dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/Template.java

```diff
@@ -44,12 +39,9 @@
     public Template() {
 
     }
-
-    //Constructor for create
     public Template(Dataverse dataverseIn) {
         dataverse = dataverseIn;
         datasetFields = initDatasetFields();
```

Comment:
`initDatasetFields()` is the same code as `DatasetFieldInitializer.createBlankDatasetFields` and `DatasetFieldInitializer.sortDatasetFieldsRecursively`. In effect we do this twice.